fastly/blobstorage_test.go

```diff
@@ -55,6 +56,7 @@ func TestClient_BlobStorages(t *testing.T) {
 			FormatVersion:   2,
 			MessageType:     "classic",
 			Placement:       "waf_debug",
+			FileMaxBytes:    10 << 20,
```

Comment:
Can we put these values into constants that can be referenced with a nice descriptive name like `fileBytesExceeded`, `fileBytesAllowed` etc. As that will help make the test code easier to understand 'at a glance'.

**Nitpick/personal preference**: I'd prefer to just use the resulting value rather than a bitwise operator to generate the value, as I now need to open a console and run the left shift to see what the resulting number is (in this case `10485760`). My ability to validate the expectations of the code is made easier when I don't have to do mental hurdle jumps.

Reply:
I created a `MiB` constant and updated the `FileMaxBytes` values to be expressed in terms of that constant.